galaxy.yml

```diff
@@ -12,7 +12,7 @@ tags:
   - vmware
   - virtualization
 dependencies:
-  cloud.common: ">=2.0.4,<3.0.0"
+  cloud.common: ">=3.0.0,<4.0.0"
```

Comment:
There's no `cloud.common` 3.0.0 at the moment, so this collection can't depend on it!

Why do you want to define this dependency. Did you test it? HOW? Without a `cloud.common` 3.0.0 available?

@alinabuzachis @jillr

Reply:
We are planning to release cloud.common 3.0.0 soon. This collection will be released after cloud.common 3.0.0 is available. We do test these collections together since vmware_rest has a dependency on cloud.common

Reply:
> We are planning to release cloud.common 3.0.0 soon. This collection will be released after cloud.common 3.0.0 is available. We do test these collections together since vmware_rest has a dependency on cloud.common

I'm afraid you're not getting the point here. You can't define a dependency on a `cloud common` release that doesn't exist (yet). Release `cloud.common` 3.0.0 and test if if this collection really works with this release. That's a prerequisite for defining it as a requirement.

No matter what you're _planing_ to do, you just can't have a requirement on non-existent collection versions.

Release `cloud.common` 3.0.0 and we can talk again. And, _please_, test if it works.

Reply:
Hi @mariolenz We do test this collection with cloud.common and it will be tested as such before this lands.  We're doing a lot of update work in parallel across all of the cloud collections the Ansible team manages right now as the core version update is a requirement for downstream certification.  Where there are cross-collection dependencies that does mean we will have a bit of a timing and coordination issue as we try to pre-stage PRs for initial visual review before other PRs have landed or dependent collections have been released.  My apologies that wasn't clear.  I believe @alinabuzachis is actively working on the cloud.common release, so @GomathiselviS was trying to stage everything here for when that lands.  I think the miscommunication could be that this PR is not marked WIP, I will change that now.

We all have the same objective of ensuring the software that we provide our users will be tested and reliable.  I would appreciate if we could all remain courteous to each other while working together towards that goal.

Reply:
I'm sorry, I guess I've overreacted a bit. But, you see, dependencies on software versions that don't exist (yet) feels so utterly _wrong_ to me...

Sorry :-/